### PR TITLE
Lesson 4, Mailroom Pt. 2

### PR DESCRIPTION
**Pull request by ganttArt**

### CODE REVIEW

File: students/christopher_gantt/lesson04/mailroom.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+DONOR_DB = {"Wassily Kandinsky": [43928.13, 131.34, 1928.0],
+            "Jasper Johns": [3134.43, 153.34],
+            "Mark Rothko": [135353.33],
+            "Richard Serra": [153757.87, 28457.12, 1293451.0],
+            "Yves Tanguy": [1534.23, 2542.19]}
```
Comment (natasha-aleksandrova) [inline students/christopher_gantt/lesson04/mailroom.py]: :thumbsup:

not that this should be `donor_db` - all caps is used for constants (things that are immutable)

File: students/christopher_gantt/lesson04/mailroom.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+
+def main():
+    print("Welcome to The Mailroom")
+    while True:
+        response = input(PROMPT)
+        if response == "4": 
+            print("Thank you for visiting The Mailroom")
+            break
+        if int(response)>4:
```
Comment (natasha-aleksandrova) [inline students/christopher_gantt/lesson04/mailroom.py]: to make this more dynamic -> `if response not in SWITCH_FUNC_DICT`

File: students/christopher_gantt/lesson04/mailroom.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+
+SWITCH_FUNC_DICT = {"1": thank_you, "2": report, "3": letter_to_all}
+
+
+def main():
+    print("Welcome to The Mailroom")
+    while True:
+        response = input(PROMPT)
+        if response == "4":
```
Comment (natasha-aleksandrova) [inline students/christopher_gantt/lesson04/mailroom.py]: to make this even cleaner you can map optin "4" to a function that call sys.exit:
```
def quit_program():
    print("Bye!")
    sys.exit()
```

File: students/christopher_gantt/lesson04/mailroom.py (hunk 1 above)
Comment (ganttArt) [reply students/christopher_gantt/lesson04/mailroom.py]: Thank you! I thought it was used for all global variables.

**Pull request merged**